github-archive/schema.sql

```diff
@@ -20,6 +20,6 @@ CREATE TABLE github (
     public BOOLEAN
 ) PARTITIONED BY (month_partition)
 WITH (
-    number_of_replicas = 0,
+    number_of_replicas = 1,
```

Comment:
the import script then should set the number of replicas before importing and reset it once it's done